docker/linux/Dockerfile

```diff
@@ -1,4 +1,16 @@
-FROM mcr.microsoft.com/dotnet/runtime-deps:6.0
+FROM debian:11-slim
+ENV ASPNETCORE_URLS=http://+:80 DOTNET_RUNNING_IN_CONTAINER=true
+RUN apt-get update && \
+    apt-get install -y --no-install-recommends \
+        ca-certificates \
+        libc6 \
+        libgcc1 \
+        libgssapi-krb5-2 \
+        libicu67 \
+        libssl1.1 \
+        libstdc++6 \
+        zlib1g && \
+    rm -rf /var/lib/apt/lists/*
```

Comment:
This uses the same original base OS image with the dependencies installed in the same manner as the Microsoft image.

Shifting away from the Microsoft image as the .NET6 EOL puts that image at high risk of being removed/out of date.